RequestService/RequestService.Handlers/GetNewsTickerHandler.cs

```diff
@@ -29,18 +32,30 @@ public async Task<NewsTickerResponse> Handle(NewsTickerRequest request, Cancella
                 Messages = new List<NewsTickerMessage>()
             };
 
-            var completedActivities = await _repository.GetCompletedActivitiesCount(request.GroupId);
-            var completedActivitiesToday = await _repository.GetActivitiesCompletedLastXDaysCount(request.GroupId, 1);
-            var requestsAddedWithinLastWeek = await _repository.GetRequestsAddedLastXDaysCount(request.GroupId, 7);
-            var openJobCount = await _repository.OpenJobCount(request.GroupId);
+            List<int> groups = new List<int>();
+
+            if(request.GroupId.HasValue)
+            {
+                groups.Add(request.GroupId.Value);
+                var childGroups = await _groupService.GetChildGroups(request.GroupId.Value);
+                groups.AddRange(childGroups.ChildGroups.Select(sm => sm.GroupId));
+            }
+
+            var completedActivities = await _repository.GetCompletedActivitiesCount(groups);
+            var completedActivitiesToday = await _repository.GetActivitiesCompletedLastXDaysCount(groups, 1);
+            var requestsAddedWithinLastWeek = await _repository.GetRequestsAddedLastXDaysCount(groups, 7);
+            var openJobCount = await _repository.OpenJobCount(groups);
 
             foreach (SupportActivityCount item in completedActivities.Where(x=> x.Value>10).OrderByDescending(x => x.Value).Take(3))
             {
+                string strRequestType = item.SupportActivity.RequestType().FriendlyName(Convert.ToInt32(item.Value));
+                string strSupportActivity = item.SupportActivity.FriendlyNameShort().ToLower();
+
                 response.Messages.Add(new NewsTickerMessage()
                 {
                     Value = item.Value,
                     SupportActivity = item.SupportActivity,
-                    Message = $"**{ item.Value } {item.SupportActivity.FriendlyNameShort().ToLower()}** { item.SupportActivity.RequestType().FriendlyName(Convert.ToInt32(item.Value)) } completed"
+                    Message = string.Format("**{0:n0} {1}** {2} completed", item.Value, strSupportActivity, strRequestType)
```

Comment:
Sorry, I know I already signed this off, but I've checked our teams chat and this can be improved further...
`Message = $"**{item.Value:n0} {strSupportActivity}** {strRequestType} completed"`